### PR TITLE
Added documentation for webhooks and document processing tutorials

### PR DESCRIPTION
**Pull request by vibehavjha**
## Category

- [ ] Content fix
- [x] New article

## Related issues

- fixes #NA
- partially #NA
- mentioned in #NA

## What's in this Pull Request?
Added new Using Webhooks and Enable Document Processing with ACS tutorials.

### CODE REVIEW

**Comment by vibehavjha** [comment]
Hi @marcwindle, please review this PR

File: docs/embedded/tutorials/doc-processing-acs.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+1.	Get the delta changes of the container. You are currently able to get the notification whenever there is any change in our container and will now get the files which are added or updated.
+2.	Call ACS’s Document Intelligence service API. You will need to create an Azure AI resource to use the API to extract the fields from an image and get the extracted files. You may store them as shown in this tutorial or you may process them as you like.
+![document processing schema](../images/Document-Processing.png)
+
+> [!TIP]
+> To learn more about the Graph APIs used in this tutorial, see [Track changes for a Drive](https://onedrive.visualstudio.com/OneDrive%20Service/_git/apidocs?path=/docs/rest-api/api/driveitem_delta.md), [Get a DriveItem resource](https://onedrive.visualstudio.com/OneDrive%20Service/_git/apidocs?path=/docs/rest-api/api/driveitem_get.md), and [Upload or replace the contents of a DriveItem](https://onedrive.visualstudio.com/OneDrive%20Service/_git/apidocs?path=/docs/rest-api/api/driveitem_put_content.md).
```
Comment (andrewconnell) [inline docs/embedded/tutorials/doc-processing-acs.md:19]: None of these links work. Must fix

File: docs/embedded/tutorials/doc-processing-acs.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
+---
+title: Document Processing with ACS
```
Comment (andrewconnell) [inline docs/embedded/tutorials/doc-processing-acs.md:2]: Spell out ACS everywhere... it's not commonly used for Azure Cognitive Services, but also easily confused with other services (including the retired Azure Access Control Service).

File: docs/embedded/tutorials/doc-processing-acs.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+description: Enabling document processing with ACS
+ms.date: 02/26/2024
+ms.localizationpriority: high
+---
+
+# Enabling Document Processing with ACS
+
+## Utilizing Azure Cognitive Services
```
Comment (andrewconnell) [inline docs/embedded/tutorials/doc-processing-acs.md:10]: Proper markdown formatting is to include a line break between each heading & paragraph. See existing docs for reference.

File: docs/embedded/tutorials/doc-processing-acs.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+1.	Get the delta changes of the container. You are currently able to get the notification whenever there is any change in our container and will now get the files which are added or updated.
+2.	Call ACS’s Document Intelligence service API. You will need to create an Azure AI resource to use the API to extract the fields from an image and get the extracted files. You may store them as shown in this tutorial or you may process them as you like.
+![document processing schema](../images/Document-Processing.png)
+
+> [!TIP]
+> To learn more about the Graph APIs used in this tutorial, see [Track changes for a Drive](https://onedrive.visualstudio.com/OneDrive%20Service/_git/apidocs?path=/docs/rest-api/api/driveitem_delta.md), [Get a DriveItem resource](https://onedrive.visualstudio.com/OneDrive%20Service/_git/apidocs?path=/docs/rest-api/api/driveitem_get.md), and [Upload or replace the contents of a DriveItem](https://onedrive.visualstudio.com/OneDrive%20Service/_git/apidocs?path=/docs/rest-api/api/driveitem_put_content.md).
+
+
+## Get the delta changes of a container
+Open `GraphProvider.ts` and implement method `getDriveChanges` to get the list of changed items.
```
Comment (andrewconnell) [inline docs/embedded/tutorials/doc-processing-acs.md:23]: Filenames should be **bold**, not inline code formatted.

File: docs/embedded/tutorials/using-webhooks.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+ms.localizationpriority: high
+---
+
+# Using Webhooks
+
+## Set Up Webhooks with SharePoint Embedded
+Webhooks are automated messages that are transmitted by an application when a trigger is activated. They can be used in SPE to enable the automation of workflows, the integration of systems, and to respond to events in real time.
+You will use webhooks to invoke the Azure Cognitive Service APIs from the application whenever an existing file is updated, or a new file is uploaded.
+To set up webhooks with your [current SharePoint Embedded application](/training/modules/sharepoint-embedded-create-app/), you need to
+1.	Create and register a webhook endpoint to get notifications whenever there is change in your container. This will be done using REST APIs.
```
Comment (andrewconnell) [inline docs/embedded/tutorials/using-webhooks.md:14]: Don't explicitly number bullets. All bullets should be `1.` and let the rendering engine create the correct numbers.

**Review by andrewconnell — CHANGES_REQUESTED** [review]

**Comment by andrewconnell** [comment]
Thanks for the new doc contribution! There are multiple things we need to address before we can fully review this. Please see my comments above. Feel free to ask questions if anything isn't clear.

**Comment by vibehavjha** [comment]
Hi @andrewconnell, I have implemented the suggested changes for both tutorials.

**Comment by marcwindle** [comment]
[like]  Marc Windle reacted to your message:
________________________________

**Comment by andrewconnell** [comment]
@vibehavjha said:
> I have implemented the suggested changes for both tutorials.

Can you review & address the automated build issue findings reported in the last build? https://github.com/SharePoint/sp-dev-docs/pull/9543#issuecomment-1991084063

Most of these are suggestions, but MSDOCs does require us to use relative links for learn content, and to not include locale's in the URLs, rather letting the MSFT gateway redirect to the appropriate locale. Also, there's a broken link reference.

**Review by andrewconnell — CHANGES_REQUESTED** [review]
Please fix absolute & broken links in the automated build report.

**Comment by vibehavjha** [comment]
> Please fix absolute & broken links in the automated build report.

I have fixed the absolute links and fixed locales. The broken links in each of the tutorials point to the other one. Please let me know if I should remove the hyperlinks and then add them back once the pages are live.

**Review by andrewconnell — APPROVED** [review]

**Comment by andrewconnell** [comment]
I made some grammatical fixes and fixed those links. When you use the leading `/file.md`, that says "go back to the root of the repo" which isn't correct and will create broken links. To stay in the same folder, you use `./file.md` which resolves both of the broken link issues.

**Comment by vibehavjha** [comment]
Thank you @andrewconnell, I'll remember the suggested changes for future articles.

**Pull request merged**